charts/fluent-operator/templates/fluent-operator-deployment.yaml

```diff
@@ -15,6 +15,9 @@ spec:
       labels:
         app.kubernetes.io/component: operator
         app.kubernetes.io/name: fluent-operator
+        {{- if .Values.operator.labels }}
+        {{- toYaml .Values.operator.labels | nindent 8 }}
```

Comment:
```suggestion
        {{- toYaml .Values.operator.labels | indent 8 }}
```

Reply:
I think the combination of `-` at the beginning and `indent` will break it and introduce errors.
In my understanding `-` at the beginning prevents helm from introducing a newline, but my `nindent 8`
will add that new line and also add the 8 spaces.  Just replacing `indent 8` with  `indent 8` will lead to errors as it would produce output like this:
```
labels:
        app.kubernetes.io/component: operator
        app.kubernetes.io/name: fluent-operatormyExampleValue0: someValue0
        myExampleValue1: someValue1
```
Above example assumes that the following labels are defined in the values.yaml

```
operator:
  labels:
    myExampleValue0: someValue0
    myExampleValue1: someValue1
```

If we do not want to use `nindent` we have to to remove `-` and put the templating statement to the very left. So I am convinced my solution above works. The alternative solution without `nindent` is this:

```
labels:
        app.kubernetes.io/component: operator
        app.kubernetes.io/name: fluent-operator
        {{- if .Values.operator.labels }}
{{ toYaml .Values.operator.labels | indent 8 }}
       {{- end }}
```